### PR TITLE
Feature/add new visit

### PR DESCRIPTION
**Pull request by sandorbanvitagroup**

### CODE REVIEW

**Review by danielrus — DISMISSED** [review]

**Review by adriandumavitagroup — DISMISSED** [review]

**Review by adriandumavitagroup — DISMISSED** [review]

**Review by danielrus — DISMISSED** [review]

File: src/app/cases/case_components/case-follow-up-add-visit/case-follow-up-add-visit-data.ts [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+const optionsTemperatureSource = pipe.transform(TemperatureSource);
+const optionsTemperature = range(35, 44.1, 0.1).map((val) => ({
+  key: Number(val.toFixed(1)),
+  value: `${val.toFixed(1)} °C`,
+}));
+
+export const FORM_DATA_CASE_FOLLOW_UP_NEW_VISITS = [
+  {
+    id: 'details',
+    title: 'visitDeatiles',
```
Comment (danielrus) [inline src/app/cases/case_components/case-follow-up-add-visit/case-follow-up-add-visit-data.ts]: spelling error (in the translation files too)

File: src/app/cases/case_components/case-follow-up-add-visit/case-follow-up-add-visit.component.ts [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+    setTimeout(() => {
+      const { form } = this.dynamicForm;
+      if (form) {
+        const availability = form.get('availability');
+        availability.setValue('COOPERATIVE');
+
+        this.subscriptions.push(
+          availability.valueChanges.subscribe((val: string) => {
+            if (val !== 'COOPERATIVE') {
```
Comment (danielrus) [inline src/app/cases/case_components/case-follow-up-add-visit/case-follow-up-add-visit.component.ts]: this if/else statement is huge and hard to follow;
one way to make it more readable could be:
- make a separate method and call it in the subscribe: availability.valueChanges.subscribe((val: string) => this.toggleSections(val === 'COOPERATIVE'))
- in the toggleSections method have an array of strings where you declare the ids of sections that needs to be updated
- iterate throught that update the group attribute (with the boolean value passed to the method)

Comment (sandorbanvitagroup) [reply src/app/cases/case_components/case-follow-up-add-visit/case-follow-up-add-visit.component.ts]: Simplified it.

File: src/app/cases/case_components/case-follow-up-add-visit/case-follow-up-add-visit-data.ts (hunk 1 above)
Comment (sandorbanvitagroup) [reply src/app/cases/case_components/case-follow-up-add-visit/case-follow-up-add-visit-data.ts]: corrected.

**Review by danielrus — DISMISSED** [review]

**Review by adriandumavitagroup — APPROVED** [review]

**Review by danielrus — APPROVED** [review]
